source/includes/Concepts/concept.md

```diff
@@ -83,7 +79,7 @@ referenced source.
     *name* | `String` | ConceptNames are the words or phrases used to express the idea of a Concept within a particular locale
     *term* | `String` | Medical coding term or OpenMRS concept dictionary term that could be mapped to a concept
     *source* | `String` | A concept can have any number of mappings to any number of other vocabularies. Other vocabularies are called "concept sources" in OpenMRS (ie. LOINC, SNOMED, ICD-9, ICD10, RxNORM, etc), but the concept source can also be a custom (ie. org.openmrs.module.mdrtb, PIH, AMPATH, MVP, etc.). Every concept can define a string for its mapping in any "concept source" defined in the database
-    *class* | `String` | The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  In this way, the class is helpful for data extraction.  This classification details how a concept will be represented (i.e. as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when defining concepts, but implementation sites may also create additional custom concept classes for use.
+    *class* | `String` | The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  In this way, the class is helpful for data extraction.  This classification elaborates how a concept will be represented (i.e., as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when defining concepts, but implementation sites may also create additional custom concept classes.
```

Comment:
```suggestion
    *class* | `String` | The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  In this way, the class is helpful for data extraction.  This classification elaborates how a concept will be represented (i.e. as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when defining concepts, but implementation sites may also create additional custom concept classes.
```

Reply:
@sanjulamadurapperuma actually its notationally correct to use a comma after i.e. Like the one I have used above If its not I will change it 🙂

Reply:
It's actually in American English. They use **i.e,** with the comma instead of the last full-stop/period. If we are okay with following that notation, then I am +1 to have **i.e,** instead of **i.e.,** which seems out-of-place.

Reply:
Okay @sanjulamadurapperuma I will make the changes 🙂